addons/purchase_mrp/models/purchase_mrp.py

```diff
@@ -11,20 +11,24 @@ class PurchaseOrderLine(models.Model):
     qty_received = fields.Float(compute='_compute_qty_received', string="Received Qty", store=True)
 
     def _compute_qty_received(self):
-        super(PurchaseOrderLine, self)._compute_qty_received()
+        phantom_lines = self.env['purchase.order.line']
```

Comment:
There was a formula that calculates the difference based on the min quantity received/ total qty for the moves related to the same purchase line by product.  Why don't we apply this here?  The calculation is less heavy to the system.